demos/notebooks/demo_pipeline.ipynb

```diff
@@ -448,13 +447,14 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-info\" markdown=\"1\">\n",
-    "    <h2 style=\"margin-top: 0;\">Optimizing performance</h2>\n",
-    "<br>    \n",
-    "If you hit memory issues later, there are a few things you can do. First, you may want to lower the number of processors you are using. Each processor uses more RAM, and on a workstation with many processors, you can sometimes get better performance by reducing <em>num_processors_to_use</em>.The best way to determine the optimal number is by trial and error. When you set <em>num_processors_to_use</em> variable to <em>None</em>, it defaults to <i>one</i> less than the total number of CPU cores available (the reason we don't automatically set it to the total number of cores is because in practice this typically leads to worse performance).<br>\n",
+    "    <h2 style=\"margin-top: 0;\">Optimizing performance</h2>  \n",
+    "    If you hit memory issues later, there are a few things you can do. First, you may want to lower the number of processors you are using. Each processor uses more RAM, and on a workstation with many processors, you can sometimes get better performance by reducing <em>num_processors_to_use</em>.The best way to determine the optimal number is by trial and error. When you set <em>num_processors_to_use</em> variable to <em>None</em>, it defaults to <i>one</i> less than the total number of CPU cores available (the reason we don't automatically set it to the total number of cores is because in practice this typically leads to worse performance).\n",
+    "\n",
+    "<br>Second, if your system has less than 32GB of RAM, and things are running slowly or you are running out of memory, then get more RAM. While you can sometimes get away with less, we recommend a *bare minimum* level of 16GB of RAM, but more is better. 32GB RAM is acceptable, 64GB or more is best. Obviously, this will depend on the size of your data sets.\n",
```

Comment:
In theory we could mention that OSX users have less ram than they think because of hybrid memory, but maybe that's too much detail to put in a notebook; good to see specific numbers here.